zerver/views/user_settings.py

```diff
@@ -109,6 +110,17 @@ def json_left_side_userlist(request, user_profile, left_side_userlist=REQ(valida
 
     return json_success(result)
 
+
+@has_request_variables
+def mark_user_star(request, user_profile,
+                           mark_star_email = REQ(validator=check_string, default=None)):
+    result = mark_user_star_backend(user_profile, mark_star_email)
+    return json_success(result)
+
+@has_request_variables
+def list_starred_user(request, user_profile):
+    return json_success(user_profile.starred_users)
```

Comment:
`user_profile.starred_users` seems to be a non-jsonable object: `django.db.models.fields.related.ManyRelatedManager`

Please add a test for `list_starred_user`